test/infrastructure/docker/exp/api/v1alpha3/groupversion_info.go

```diff
@@ -14,9 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha3 contains API Schema definitions for the exp.infrastructure v1alpha3 API group
+// Package v1alpha3 contains API Schema definitions for the infrastructure v1alpha3 API group
 // +kubebuilder:object:generate=true
-// +groupName=exp.infrastructure.cluster.x-k8s.io
+// +groupName=infrastructure.cluster.x-k8s.io
```

Comment:
Q: is this a breaking change or should we consider it a bug from recent group change? Same for v1alpha4

@CecileRobertMichon ^^

Reply:
IMO we should mark the change as breaking, especially given https://github.com/kubernetes-sigs/cluster-api/pull/5206/files#r702642637 (we have to rename one of the types) but because this is fixing a bug in the exp/ types I think it's okay to release it as an experimental breaking change

@vincepri thoughts?

Reply:
+1, should be fine. We should probably still mark the PR as ⚠️

Reply:
Changed the label to ⚠️